daemon/execdriver/lxc/driver.go

```diff
@@ -85,16 +85,21 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 		dataPath = d.containerDir(c.ID)
 	)
 
+	container, err := d.createContainer(c)
```

Comment:
To be honest, I'm not familiar with this code but before this PR it would set the input variable 'c.ProcessConfig.Termal' to 'true' *before* calling createContainer. But this PR sets that input variable *after* calling createContainer, which I would think would have no effect.  This seems backwards.  Am I missing something?